wellcad/com/_borehole.py

```diff
@@ -2384,7 +2461,8 @@ def cased_hole_ultrasonics(self, abiwavelet_log=None, zone_log=None, prompt_user
         self._dispatch.CasedHoleUltrasonics(abiwavelet_log, zone_log, prompt_user, config)
 
     def calculate_apparent_metal_loss(self, log=None, prompt_user=None, config=None):
-        """Calculates an apparent metal loss value for each trace of radius values stored in an image log.
+        """This function is deprecated in WellCAD 5.7 and onwards. Please use calculate_apparent_metal_loss_ex instead.
```

Comment:
Best that the deprecation notice comes after the first line description. Should we be using the official Sphinx deprecation notice?

https://numpydoc.readthedocs.io/en/latest/format.html#deprecation-warning